src/lib/probe/crates/probe-quota/CLAUDE.md

```
@@ -0,0 +1,31 @@
+<!-- updated: 2026-02-15T21:30:00Z -->
+# probe-quota - Resource Quota Management
+
+Cross-platform resource quota detection and management.
+
+## Structure
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)